packages/salesforcedx-vscode-core/src/commands/forceCreateManifest.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
```

Comment:
Can the year be changed to 2021?

Reply:
Yes!

Reply:
I'm stuck in 2021. I meant 2022 :)

Reply:
HA!  I had 2021 in my latest PR as well.  (I just updated them)

Reply:
I'm glad you clarified, because I'll also be in 2021 until at least February.

Reply:
Are we using https://marketplace.visualstudio.com/items?itemName=doi.fileheadercomment for auto inserting the copyright? looks like there is a year template value if so.

Reply:
@gbockus-sf - We don't use it. We just copy it from other files. This extension is a good idea for the whole team to use 👍

Reply:
cool I'll create a ticket and follow up with adding it to our suggested extensions.